app/assets/stylesheets/layouts.scss

```diff
@@ -16,3 +16,20 @@
   width: 250px;
   margin-left: 40px;
 }
+
+// SIGN OUT LINK NEEDS TO BE A BUTTON_TO. STYLES MATCH BOOTSTRAP
+.navbar #sign_out_nav_link {
```

Comment:
yeah, it was already in place. i'll just leave it on this branch because on redoing the frontend, i plan to remove all of these ids that i believe were added for tests.